### ISSUE TITLE
[Config Support]: Failed to start add-on Port '8555'

### ISSUE THREAD

**Issue by ronrath** [blakeblackshear/frigate#5971]
### Describe the problem you are having

I just upgraded to 0.12.0 and Frigate won't start.  When I click on "Start" I get the following message:  "Failed to start add-on
Port '8555' is already in use by something else on the host."  Nothing in the log file since it didn't try to start.  Everything was working fine prior to upgrade.  I tried repotting my host and that did not fix the problem.

### Version

0.12.0

### Frigate config file

```yaml
mqtt:
  host: 192.168.1.112
  port: 1883
  topic_prefix: frigate
  client_id: frigate
  user: homeassistant
  password: 
  stats_interval: 60

database:
  path: /media/frigate/clips/frigate.db
  
detectors:
  coral:
    type: edgetpu
    device: usb

cameras:
  garage:
    ffmpeg:
      input_args: -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -stimeout 5000000 -use_wallclock_as_timestamps 1
      output_args: 
        record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
      inputs:
        - path: rtsp://user:pwd@192.168.70.72:554/cam/realmonitor?channel=1&subtype=1
          roles:
            - detect
            - rtmp
        - path: rtsp://user:pwd@192.168.70.72:554/live
          roles:
            - record
    detect:
      width: 704
      height: 480
      fps: 5
    objects:
      track:
        - person
        - cat
      filters:
        person:
          threshold: 0.6
    motion:
      mask:
        - 700,0,700,46,515,46,515,0
        - 704,480,704,457,446,313,269,480
        - 704,192,641,140,512,123,500,202,704,354
    record:
      enabled: True
      retain:
        days: 0
      events:
        retain:
          default: 10
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: True
      retain:
        default: 5
        
  basement:
    ffmpeg:
      input_args: -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -stimeout 5000000 -use_wallclock_as_timestamps 1
      output_args: 
        record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac      
      inputs:
        - path: rtsp://usedr:pwd@192.168.70.71:554/cam/realmonitor?channel=1&subtype=1
          roles:
            - detect
            - rtmp
        - path: rtsp://user:pwd@192.168.70.71:554/live
          roles:
            - record
    detect:
      width: 704
      height: 480
      fps: 5
    objects:
      track:
        - person
        - cat
      filters:
        person:
          threshold: 0.6
    motion:
      mask:
        - 700,0,700,46,515,46,515,0
    record:
      enabled: True
      retain:
        days: 0
      events:
        retain:
          default: 10
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: True
      retain:
        default: 5
        
  backyard:
    ffmpeg:
      input_args: -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -stimeout 5000000 -use_wallclock_as_timestamps 1
      output_args: 
        record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
      inputs:
        - path: rtsp://user:pwd@192.168.70.73:554/cam/realmonitor?channel=1&subtype=1
          roles:
            - detect
            - rtmp
        - path: rtsp://user:pwd@192.168.70.73:554/live
          roles:
            - record
    detect:
      width: 704
      height: 480
      fps: 5
    objects:
      track:
        - person
        - cat
      filters:
        person:
          threshold: 0.6
    motion:
      mask:
        - 700,0,700,46,515,46,515,0
    record:
      enabled: True
      retain:
        days: 0
      events:
        retain:
          default: 10
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: True
      retain:
        default: 5

  front:
    ffmpeg:
      input_args: -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -stimeout 5000000 -use_wallclock_as_timestamps 1
      output_args: 
        record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
      inputs:
        - path: rtsp://user:pwd@192.168.70.74:554/cam/realmonitor?channel=1&subtype=1
          roles:
            - detect
            - rtmp
        - path: rtsp://user:pwd@192.168.70.74:554/live
          roles:
            - record
    detect:
      width: 704
      height: 480
      fps: 5
    objects:
      track:
        - person
        - cat
      filters:
        person:
          threshold: 0.6
    motion:
      mask:
        - 700,0,700,46,515,46,515,0
    record:
      enabled: True
      retain:
        days: 0
      events:
        retain:
          default: 10
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: True
      retain:
        default: 5
        
  front-door:
    ffmpeg:
      input_args: -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -stimeout 5000000 -use_wallclock_as_timestamps 1
      output_args: 
        record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
      inputs:
        - path: rtsp://user:pwd@192.168.70.75:554/cam/realmonitor?channel=1&subtype=1
          roles:
            - detect
            - rtmp
        - path: rtsp://user:pwd@192.168.70.75:554/live
          roles:
            - record
    detect:
      width: 704
      height: 480
      fps: 5
    objects:
      track:
        - person
        - cat
      filters:
        person:
          threshold: 0.6
    motion:
      mask:
        - 700,0,700,46,515,46,515,0
    record:
      enabled: True
      retain:
        days: 0
      events:
        retain:
          default: 10
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: True
      retain:
        default: 5
                
birdseye:
  enabled: True
  width: 1280
  height: 720
  # Optional: Encoding quality of the mpeg1 feed (default: shown below)
  # 1 is the highest quality, and 31 is the lowest. Lower quality feeds utilize less CPU resources.
  quality: 3
  # Optional: Mode of the view. Available options are: objects, motion, and continuous
  #   objects - cameras are included if they have had a tracked object within the last 30 seconds
  #   motion - cameras are included if motion was detected in the last 30 seconds
  #   continuous - all cameras are included always
  mode: motion
```

### Relevant log output

```shell
n/a
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

USB

**Comment by ronrath** [comment]
I deleted all the cards and still had the problem.  I then disabled the WebRTC integration and was able to get past the port 8555 error but now I get this:

s6-rc: info: service s6rc-fdholder: starting
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service s6rc-fdholder successfully started
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service log-prepare: starting
s6-rc: info: service log-prepare successfully started
s6-rc: info: service nginx-log: starting
s6-rc: info: service go2rtc-log: starting
s6-rc: info: service frigate-log: starting
s6-rc: info: service go2rtc-log successfully started
s6-rc: info: service go2rtc: starting
s6-rc: info: service nginx-log successfully started
s6-rc: info: service frigate-log successfully started
s6-rc: info: service go2rtc successfully started
s6-rc: info: service go2rtc-healthcheck: starting
s6-rc: info: service frigate: starting
s6-rc: info: service go2rtc-healthcheck successfully started
s6-rc: info: service frigate successfully started
s6-rc: info: service nginx: starting
2023-04-10 16:23:42.812688753  [INFO] Preparing go2rtc config...
2023-04-10 16:23:42.813904190  [INFO] Starting Frigate...
s6-rc: info: service nginx successfully started
s6-rc: info: service legacy-services: starting
2023-04-10 16:23:42.850475600  [INFO] Starting NGINX...
s6-rc: info: service legacy-services successfully started
2023-04-10 16:23:42.963545460  [INFO] Got IP address from supervisor: IP
2023-04-10 16:23:43.100538591  [INFO] Got WebRTC port from supervisor: 8555
2023-04-10 16:23:43.848402063  RuntimeError: module compiled against API version 0x10 but this version of numpy is 0xf
2023-04-10 16:23:43.848892478  Traceback (most recent call last):
2023-04-10 16:23:43.849033827    File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
2023-04-10 16:23:43.849876383      return _run_code(code, main_globals, None,
2023-04-10 16:23:43.850011454    File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
2023-04-10 16:23:43.850581035      exec(code, run_globals)
2023-04-10 16:23:43.850708754    File "/opt/frigate/frigate/__main__.py", line 9, in <module>
2023-04-10 16:23:43.851172689      from frigate.app import FrigateApp
2023-04-10 16:23:43.851295057    File "/opt/frigate/frigate/app.py", line 17, in <module>
2023-04-10 16:23:43.851772491      from frigate.comms.dispatcher import Communicator, Dispatcher
2023-04-10 16:23:43.851903840    File "/opt/frigate/frigate/comms/dispatcher.py", line 9, in <module>
2023-04-10 16:23:43.852378793      from frigate.config import FrigateConfig
2023-04-10 16:23:43.852506550    File "/opt/frigate/frigate/config.py", line 9, in <module>
2023-04-10 16:23:43.852958485      import matplotlib.pyplot as plt
2023-04-10 16:23:43.853087630    File "/usr/local/lib/python3.9/dist-packages/matplotlib/__init__.py", line 113, in <module>
2023-04-10 16:23:43.853765338      from . import _api, _version, cbook, _docstring, rcsetup
2023-04-10 16:23:43.853896742    File "/usr/local/lib/python3.9/dist-packages/matplotlib/rcsetup.py", line 27, in <module>
2023-04-10 16:23:43.854400528      from matplotlib.colors import Colormap, is_color_like
2023-04-10 16:23:43.854525062    File "/usr/local/lib/python3.9/dist-packages/matplotlib/colors.py", line 56, in <module>
2023-04-10 16:23:43.855091458      from matplotlib import _api, _cm, cbook, scale
2023-04-10 16:23:43.855220233    File "/usr/local/lib/python3.9/dist-packages/matplotlib/scale.py", line 22, in <module>
2023-04-10 16:23:43.855713945      from matplotlib.ticker import (
2023-04-10 16:23:43.855839090    File "/usr/local/lib/python3.9/dist-packages/matplotlib/ticker.py", line 138, in <module>
2023-04-10 16:23:43.856563778      from matplotlib import transforms as mtransforms
2023-04-10 16:23:43.856696924    File "/usr/local/lib/python3.9/dist-packages/matplotlib/transforms.py", line 49, in <module>
2023-04-10 16:23:43.857229449      from matplotlib._path import (
2023-04-10 16:23:43.857489296  ImportError: numpy.core.multiarray failed to import
2023-04-10 16:23:44.028267507  [INFO] Starting go2rtc...
2023-04-10 16:23:44.059697915  [INFO] Service Frigate exited with code 1 (by signal 0)
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service nginx: stopping
s6-rc: info: service go2rtc-healthcheck: stopping
2023-04-10 16:23:44.093397052  [INFO] The go2rtc-healthcheck service exited with code 256 (by signal 15)
s6-rc: info: service go2rtc-healthcheck successfully stopped
2023-04-10 16:23:44.223786898  [INFO] Service NGINX exited with code 0 (by signal 0)
s6-rc: info: service nginx successfully stopped
s6-rc: info: service nginx-log: stopping
s6-rc: info: service frigate: stopping
s6-rc: info: service frigate successfully stopped
s6-rc: info: service go2rtc: stopping
s6-rc: info: service frigate-log: stopping
s6-rc: info: service nginx-log successfully stopped
s6-rc: info: service frigate-log successfully stopped
2023-04-10 16:23:44.252129908  [INFO] The go2rtc service exited with code 256 (by signal 15)
s6-rc: info: service go2rtc successfully stopped
s6-rc: info: service go2rtc-log: stopping
s6-rc: info: service go2rtc-log successfully stopped
s6-rc: info: service log-prepare: stopping
s6-rc: info: service s6rc-fdholder: stopping
s6-rc: info: service log-prepare successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service s6rc-fdholder successfully stopped
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped

**Comment by NickM-27** [comment]
what device are you running on? is it an arm device with a 32bit OS?

**Comment by ronrath** [comment]
RPi4

**Comment by ronrath** [comment]
32. (armv7l)

**Comment by NickM-27** [comment]
recommendation is to update to 64 bit

**Comment by ronrath** [comment]
Okay.  I don't really have time to do that now so I've just restored back to 0.11.0 and everything is back up and running so I'll just stay there for now.  Thanks.

**Comment by NickM-27** [comment]
> I have the same problem. So I have to replace WebRtc for go2rtc? I was using WebRtc because of PTZ...

webrtc card uses go2rtc behind the scenes. So you simply remove the integration and then re-add it pointing the integration at frigate:1984

**Comment by rucanunes** [comment]
Could you please be more specific? How do I do that?

And what about these cards?

```
type: custom:webrtc-camera
show_state: true
show_name: true
camera_view: auto
url: rtsp://admin:xxxxxxxxxxxxx@192.168.0.203:554/Streaming/Channels/101
intersection: 0.75
ui: true
ptz:
  opacity: 0.8
  service: onvif.ptz
  distance: 0.1
  speed: 0.8
  move_mode: RelativeMove
  data_left:
    pan: LEFT
    device_id: 59b6890852a183a12d10985a1502aad6
  data_right:
    pan: RIGHT
    device_id: 59b6890852a183a12d10985a1502aad6
  data_up:
    tilt: UP
    device_id: 59b6890852a183a12d10985a1502aad6
  data_down:
    tilt: DOWN
    device_id: 59b6890852a183a12d10985a1502aad6
  data_zoom_in:
    zoom: ZOOM_IN
    entity_id: 59b6890852a183a12d10985a1502aad6
  data_zoom_out:
    zoom: ZOOM_OUT
    entity_id: 59b6890852a183a12d10985a1502aad6
```

**Comment by NickM-27** [comment]
> Could you please be more specific? How do I do that?
> 
> And what about these cards?
> 
> Thank you!
> 
> ```
> type: custom:webrtc-camera
> show_state: true
> show_name: true
> camera_view: auto
> url: rtsp://admin:xxxxxxxxxxxxx@192.168.0.203:554/Streaming/Channels/101
> intersection: 0.75
> ui: true
> ptz:
>   opacity: 0.8
>   service: onvif.ptz
>   distance: 0.1
>   speed: 0.8
>   move_mode: RelativeMove
>   data_left:
>     pan: LEFT
>     device_id: 59b6890852a183a12d10985a1502aad6
>   data_right:
>     pan: RIGHT
>     device_id: 59b6890852a183a12d10985a1502aad6
>   data_up:
>     tilt: UP
>     device_id: 59b6890852a183a12d10985a1502aad6
>   data_down:
>     tilt: DOWN
>     device_id: 59b6890852a183a12d10985a1502aad6
>   data_zoom_in:
>     zoom: ZOOM_IN
>     entity_id: 59b6890852a183a12d10985a1502aad6
>   data_zoom_out:
>     zoom: ZOOM_OUT
>     entity_id: 59b6890852a183a12d10985a1502aad6
> ```

you simply go to `Settings -> Devices & Services -> webRTC Camera` and press the 3 dots then delete. Then you press the add button, webRTC Camera and then add it. When it asks for the location of the server it will either have frigate autofilled if you run the addon or you will put frigate_ip:1984 making sure that frigate_ip is actually the ip address of frigate

**Comment by ronrath** [comment]
Upgraded to 64-bit, deleted and reinstalled webRTC, deleted all camera cards, and no change.  Same error message.

Also, no log to share since it never gets to trying to start

**Comment by NickM-27** [comment]
there will definitely be logs in the addon logs section

**Comment by Akinetopsia1** [comment]
> > Could you please be more specific? How do I do that?
> > And what about these cards?
> > Thank you!
> > ```
> > type: custom:webrtc-camera
> > show_state: true
> > show_name: true
> > camera_view: auto
> > url: rtsp://admin:xxxxxxxxxxxxx@192.168.0.203:554/Streaming/Channels/101
> > intersection: 0.75
> > ui: true
> > ptz:
> >   opacity: 0.8
> >   service: onvif.ptz
> >   distance: 0.1
> >   speed: 0.8
> >   move_mode: RelativeMove
> >   data_left:
> >     pan: LEFT
> >     device_id: 59b6890852a183a12d10985a1502aad6
> >   data_right:
> >     pan: RIGHT
> >     device_id: 59b6890852a183a12d10985a1502aad6
> >   data_up:
> >     tilt: UP
> >     device_id: 59b6890852a183a12d10985a1502aad6
> >   data_down:
> >     tilt: DOWN
> >     device_id: 59b6890852a183a12d10985a1502aad6
> >   data_zoom_in:
> >     zoom: ZOOM_IN
> >     entity_id: 59b6890852a183a12d10985a1502aad6
> >   data_zoom_out:
> >     zoom: ZOOM_OUT
> >     entity_id: 59b6890852a183a12d10985a1502aad6
> > ```
> 
> you simply go to `Settings -> Devices & Services -> webRTC Camera` and press the 3 dots then delete. Then you press the add button, webRTC Camera and then add it. When it asks for the location of the server it will either have frigate autofilled if you run the addon or you will put frigate_ip:1984 making sure that frigate_ip is actually the ip address of frigate


Deleted WebRTC, restarted, frigate still won't start. 

> Port '8555' is already in use by something else on the host

**Comment by NickM-27** [comment]
Well it's exactly the same as I said above, go2rtc uses the same ports so of course you can't run both on the same port. Frigates internal go2rtc supports all features so there's no reason to run both.

**Comment by Akinetopsia1** [comment]
> Frigates internal go2rtc supports all features so there's no reason to run both.

Ah, does it support two-way audio ?

**Comment by NickM-27** [comment]
> > Frigates internal go2rtc supports all features so there's no reason to run both.
> 
> Ah, does it support two-way audio ? 

Yes, of course

**Comment by Akinetopsia1** [comment]
Hrm, i had go2rtc in parallel to the previous version of frigate because i wanted both the motion detection and the ability to use 2-way audio via a webrtc card, but now it seems i'll have to redo my customizations.

I disabled go2rtc and attempting to access the go2rtc dashboard in the top right of frigate's system section, i get a 404. Moreover my cards stop working with a connect call failed on port 1984 error

Sounds like i'm probably going to have to uninstall both and only reinstall frigate.

**Comment by NickM-27** [comment]
> I disabled go2rtc and attempting to access the go2rtc dashboard in the top right of frigate's system section, i get a 404.

the dashboard is not supported via frigate ingress, you need to use the port to access it

> Moreover my cards stop working with a connect call failed on port 1984 error
> 
> Sounds like i'm probably going to have to uninstall both and only reinstall frigate and see i can get that working

You just need to simply remove the webrtc card integration and then add it back pointing at frigate_ip:1984 (making sure the port is open)

**Comment by ronrath** [comment]
I'm still getting the following errors during startup when I upgrade to .12   

Everything work in .11

Home Assistant Core

Error fetching information from http://ccab4aaf-frigate:5000/api/stats: Cannot connect to host ccab4aaf-frigate:5000 ssl:default [Connect call failed ('172.30.33.3', 5000)]
1:20:11 PM – (ERROR) Frigate (custom integration) - message first occurred at 1:15:56 PM and shows up 7 times
Failed to to call /addons/ccab4aaf_frigate/start - Port '8555' is already in use by something else on the host.
1:17:17 PM – (ERROR) Home Assistant Supervisor - message first occurred at 1:17:00 PM and shows up 2 times

**Comment by NickM-27** [comment]
You are probably running webrtc card integration. You should disable that, upgrade frigate, and then re-setup the webrtc card integration

**Comment by ronrath** [comment]
I've disabled the webrtc card and get the following log on startup

s6-rc: info: service s6rc-fdholder: starting
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service s6rc-fdholder successfully started
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service log-prepare: starting
s6-rc: info: service log-prepare successfully started
s6-rc: info: service nginx-log: starting
s6-rc: info: service go2rtc-log: starting
s6-rc: info: service frigate-log: starting
s6-rc: info: service nginx-log successfully started
s6-rc: info: service frigate-log successfully started
s6-rc: info: service go2rtc-log successfully started
s6-rc: info: service go2rtc: starting
s6-rc: info: service go2rtc successfully started
s6-rc: info: service go2rtc-healthcheck: starting
s6-rc: info: service frigate: starting
s6-rc: info: service go2rtc-healthcheck successfully started
s6-rc: info: service frigate successfully started
s6-rc: info: service nginx: starting
2023-05-24 13:13:06.341195633  [INFO] Preparing go2rtc config...
2023-05-24 13:13:06.345474608  [INFO] Starting Frigate...
s6-rc: info: service nginx successfully started
2023-05-24 13:13:06.361241278  [INFO] Starting NGINX...
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
2023-05-24 13:13:06.492558183  [INFO] Got IP address from supervisor: 192.168.1.112
2023-05-24 13:13:06.629623666  [INFO] Got WebRTC port from supervisor: 8555
2023-05-24 13:13:07.406732594  RuntimeError: module compiled against API version 0x10 but this version of numpy is 0xf
2023-05-24 13:13:07.407268195  Traceback (most recent call last):
2023-05-24 13:13:07.407469321    File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
2023-05-24 13:13:07.408181530      return _run_code(code, main_globals, None,
2023-05-24 13:13:07.408400860    File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
2023-05-24 13:13:07.408924516      exec(code, run_globals)
2023-05-24 13:13:07.409136975    File "/opt/frigate/frigate/__main__.py", line 9, in <module>
2023-05-24 13:13:07.409591745      from frigate.app import FrigateApp
2023-05-24 13:13:07.409786186    File "/opt/frigate/frigate/app.py", line 17, in <module>
2023-05-24 13:13:07.410234011      from frigate.comms.dispatcher import Communicator, Dispatcher
2023-05-24 13:13:07.410431655    File "/opt/frigate/frigate/comms/dispatcher.py", line 9, in <module>
2023-05-24 13:13:07.410890850      from frigate.config import FrigateConfig
2023-05-24 13:13:07.411078458    File "/opt/frigate/frigate/config.py", line 9, in <module>
2023-05-24 13:13:07.411524135      import matplotlib.pyplot as plt
2023-05-24 13:13:07.411787019    File "/usr/local/lib/python3.9/dist-packages/matplotlib/__init__.py", line 113, in <module>
2023-05-24 13:13:07.412425581      from . import _api, _version, cbook, _docstring, rcsetup
2023-05-24 13:13:07.412621522    File "/usr/local/lib/python3.9/dist-packages/matplotlib/rcsetup.py", line 27, in <module>
2023-05-24 13:13:07.413118883      from matplotlib.colors import Colormap, is_color_like
2023-05-24 13:13:07.413310175    File "/usr/local/lib/python3.9/dist-packages/matplotlib/colors.py", line 56, in <module>
2023-05-24 13:13:07.413859128      from matplotlib import _api, _cm, cbook, scale
2023-05-24 13:13:07.414056384    File "/usr/local/lib/python3.9/dist-packages/matplotlib/scale.py", line 22, in <module>
2023-05-24 13:13:07.414535505      from matplotlib.ticker import (
2023-05-24 13:13:07.414729631    File "/usr/local/lib/python3.9/dist-packages/matplotlib/ticker.py", line 138, in <module>
2023-05-24 13:13:07.415440617      from matplotlib import transforms as mtransforms
2023-05-24 13:13:07.415675724    File "/usr/local/lib/python3.9/dist-packages/matplotlib/transforms.py", line 49, in <module>
2023-05-24 13:13:07.416211603      from matplotlib._path import (
2023-05-24 13:13:07.416504357  ImportError: numpy.core.multiarray failed to import
2023-05-24 13:13:07.572692762  [INFO] Starting go2rtc...
2023-05-24 13:13:07.600866755  [INFO] Service Frigate exited with code 1 (by signal 0)
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service nginx: stopping
s6-rc: info: service go2rtc-healthcheck: stopping
2023-05-24 13:13:07.637490239  [INFO] The go2rtc-healthcheck service exited with code 256 (by signal 15)
s6-rc: info: service go2rtc-healthcheck successfully stopped
2023-05-24 13:13:07.711549600  [INFO] Service NGINX exited with code 0 (by signal 0)
s6-rc: info: service nginx successfully stopped
s6-rc: info: service nginx-log: stopping
s6-rc: info: service frigate: stopping
s6-rc: info: service nginx-log successfully stopped
s6-rc: info: service frigate successfully stopped
s6-rc: info: service go2rtc: stopping
s6-rc: info: service frigate-log: stopping
s6-rc: info: service frigate-log successfully stopped
2023-05-24 13:13:07.743384359  [INFO] The go2rtc service exited with code 256 (by signal 15)
s6-rc: info: service go2rtc successfully stopped
s6-rc: info: service go2rtc-log: stopping
s6-rc: info: service go2rtc-log successfully stopped
s6-rc: info: service log-prepare: stopping
s6-rc: info: service s6rc-fdholder: stopping
s6-rc: info: service log-prepare successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service s6rc-fdholder successfully stopped
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped

**Comment by NickM-27** [comment]
are you running on a 32 bit OS?

**Comment by ronrath** [comment]
no, 64

**Comment by ronrath** [comment]
pi@raspberrypi:~ $ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
NAME="Raspbian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
pi@raspberrypi:~ $ uname -m
aarch64

**Comment by NickM-27** [comment]
Oh, so you're not running HA OS like the issue says. In that case, I'd suggest updating docker as some users found older versions of docker caused problems.

**Comment by ronrath** [comment]
Sorry about that.  I think I have the latest version of docker

Client: Docker Engine - Community
 Version:           24.0.1
 API version:       1.43
 Go version:        go1.20.4
 Git commit:        6802122
 Built:             Fri May 19 18:06:18 2023
 OS/Arch:           linux/arm
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          24.0.1
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.20.4
  Git commit:       463850e
  Built:            Fri May 19 18:06:18 2023
  OS/Arch:          linux/arm
  Experimental:     false
 containerd:
  Version:          1.6.21
  GitCommit:        3dce8eb055cbb6872793272b4f20ed16117344f8
 runc:
  Version:          1.1.7
  GitCommit:        v1.1.7-0-g860f061
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0

**Comment by PioKow2023** [comment]
I had the identical error after the last update. In my case there was a conflict with go2rtc. After disabling go2rtc Frigate started without a problem.

**Comment by janis-vitols** [comment]
I had even dummier case. I had been playing around by installing Frigate and also Frigate (full access). When I decided that I will be using only Frigate (full access) - I uninstalled original Frigate (w/o full access) add-on. But then I couldn't start Frigate (full access) as HassOS coulnd't find docker image or something like that.

So I installed previously removed Frigate add-on and was mostly playing around with Frigate (full access). After few days I couldn't manage to start add-on as it was throwing this error (about used port).

Found out that Frigate add-on had "Start on boot" checked (turned on). So looks like after reboot 1st add-on (which I don't use) started and I couldn't start Frigate (full access).

Leaving this comment here, maybe someone will do something dummy like me and will try to figure out what's wrong 😆
